jetty-ee10/jetty-ee10-glassfish-jstl/src/test/java/org/eclipse/jetty/ee10/jstl/JstlTest.java

```diff
@@ -39,7 +39,7 @@
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.not;
 
-@Disabled //TODO
+@Disabled //Need jsp version fix to re-enable
```

Comment:
Is there a link to the issue tracking this note?

Reply:
@joakime no, it will be fixed soon by the branch jetty-12.0.x-update-jakarta-api-versions.